purchases/src/test/java/com/revenuecat/purchases/backend_integration_tests/BaseBackendIntegrationTest.kt

```diff
@@ -72,6 +72,7 @@ internal abstract class BaseBackendIntegrationTest {
     ) {
         appConfig = mockk<AppConfig>().apply {
             every { baseURL } returns URL("https://api.revenuecat.com")
+            every { paywallEventsURL } returns URL("https://api-paywalls.revenuecat.com/")
```

Comment:
This could use `AppConfig` instead of duplicating it?

Reply:
Yeah, just have to expose it, but might be better.